genesyscloud/knowledge_label/resource_genesyscloud_knowledge_label_test.go

```diff
@@ -134,3 +134,16 @@ func testVerifyKnowledgeLabelDestroyed(state *terraform.State) error {
 	// Success. All knowledge labels destroyed
 	return nil
 }
+
+func GenerateKnowledgeKnowledgebaseResource(
+	resourceLabel string,
+	name string,
+	description string,
+	coreLanguage string) string {
+	return fmt.Sprintf(`resource "genesyscloud_knowledge_knowledgebase" "%s" {
```

Comment:
Once we refactor `genesyscloud_knowledge_knowledgebase` outside of `gcloud` package we'll be able to reference this function in that package.